workspaces/icecap-runtime/Makefile

```diff
@@ -422,3 +422,4 @@ build: elfloader
 
 .PHONY: clean
 	rm -rf css-icecap.bin $(build_dir)
```

Comment:
I changed this on `main` recently. You'll probably have to rebase.

Reply:
Fixed in rebase